features/editor_can_delete_articles.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Editor can delete articles
+
+    As an editor,
+    In order to remove incorrect or inappropriate articles
+    I would like to be able to delete them
+
+    Background:
+        Given the following user exists
+            | name | email         | password | role |
+            | Jim  | jim@email.com | password | 2    |
+        And the following article exists:
+            | title      | description     | content        | user_id | category_id |
+            | It's a win | Local team wins | We didn't lose | Jim     | Sports      |
+
+    Scenario: Editor can delete article
+        Given I am logged in as 'Jim'
+        And I click on "It's a win"
```

Comment:
Use single quotes

Reply:
We need double quotes for string implementation

Reply:
I don't see you using string interpolation

Reply:
Maybe not string interpolation but we need double quotes for the comma in "It's a win"

Reply:
okey!